Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_First_SSO_Access.yml

```diff
@@ -0,0 +1,1906 @@
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "111"
+    scriptarguments:
+      Username:
+        complex:
+          root: inputs.Username
+          transformers:
+          - operator: Cut
+            args:
+              delimiter:
+                value:
+                  simple: \
+              fields:
+                value:
+                  simple: "2"
+    separatecontext: true
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": -1570
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 2
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "100":
+    id: "100"
+    taskid: 670ae50d-528f-40c5-8f9b-5976a8ade4f5
+    type: collection
+    task:
+      id: 670ae50d-528f-40c5-8f9b-5976a8ade4f5
+      version: -1
+      name: Get Manager Response
+      description: |-
+        Ask for the user's manager.
+        Did the user work from the country? (The country from the incident.)
+      type: collection
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "155"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 20,
+          "y": 1420
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    message:
+      to:
+        complex:
+          root: UserManagerEmail
+      subject:
+        simple: Did the user ${Account.DisplayName} work from ${incident.countryname}?
+      body: {}
+      methods:
+      - email
+      format: ""
+      bcc:
+      cc:
+      timings:
+        retriescount: 2
+        retriesinterval: 360
+        completeafterreplies: 1
+        completeafterv2: true
+        completeaftersla: false
+    form:
+      questions:
+      - id: "0"
+        label: ""
+        labelarg:
+          simple: Did the user ${Account.DisplayName} work from ${incident.countryname}?
+        required: true
+        gridcolumns: []
+        defaultrows: []
+        type: singleSelect
+        options: []
+        optionsarg:
+        - simple: "Yes"
+        - simple: "No"
+        fieldassociated: ""
+        placeholder: ""
+        tooltip: ""
+        readonly: false
+      title: Did the user ${Account.DisplayName} work from ${incident.countryname}?
+      description: ""
+      sender: ""
+      expired: false
+      totalanswers: 0
+    skipunavailable: false
+    quietmode: 2
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "110":
+    id: "110"
+    taskid: 9f56f01f-eed3-49b8-8f4c-587385bb735f
+    type: playbook
+    task:
+      id: 9f56f01f-eed3-49b8-8f4c-587385bb735f
+      version: -1
+      name: Cortex XDR - First SSO Investigation
+      description: |
```

Comment:
What is that playbook??

Reply:
Done.
the playbook was removed. task moved to main palybook